README.md

```diff
@@ -90,6 +90,28 @@ docker run --rm omisego/plasma-contracts npx truffle migrate --network remote
 
 For more detail of the deploying scripts and the env vars to set, see [deploying.md](./plasma_framework/docs/deploying/deploying.md)
 
+The `MAINTAINER` account can also be a multisignature contract. The example we've built uses Gnosis MultiSigWallet. The project has been added as a git submodule:
```

Comment:
can we give a link to their repo too in this section? https://github.com/gnosis/MultiSigWallet/tree/v1.6.0